docs/extensions/my-first-extension/_posts/1970-01-04-CreatingShortcutAndScreen.md

```diff
@@ -122,52 +81,71 @@ export const themes = extension.themes;
+
+We already did this in Getting Started, but let's elaborate on this. Since the app is managed through the Builder, we needed to `push` the extension to Shoutem and `install` it into our app so we can use and preview it in the Builder:
 
 ```ShellSession
 $ shoutem push
 Uploading `Restaurants` extension to Shoutem...
 Success!
-```
-
-Open the `Add screen` modal in the Builder and select the `Custom` category. You can now see your `Restaurants` starting screen (the shortcut).
+$ shoutem install
 
-<p class="image">
-<img src='{{ site.url }}/img/my-first-extension/add-modal-shortcut.png'/>
-</p>
+Extension installed.
+See it in the builder: {{ site.shoutem.builderURL }}/app/{{ site.example.appId }}
+```
 
-> #### Note
-> In case you don't see it, refresh the page in browser.
+After installing it, we opened the app in the Builder and added the extension's screen to Main navigation. Installing new extensions and adding their shortcuts to the app requires you to rebuild your local clone:
 
-Click on `Restaurants` to insert that shortcut into your app's navigation.
+```ShellSession
+$ shoutem reconfigure
 
-Let's preview your app now. We can preview it in the Builder, but it might take some time while the app preview shows. Every time you change an extension, we need to rebundle the whole app to the new extension. It's much faster to use **Shoutem Preview** app (available for [iOS]({{ site.shoutem.previewAppiOS }}) and [Android]({{ site.shoutem.previewAppAndroid }})) and Shoutem CLI, which can bundle only the changes made in the extension.
+> @shoutem/mobile-app@1.1.0 configure /path/to/Restaurants_{{ site.example.appId }}
+> node scripts/configure
+...
+```
 
-Since the app is managed through the Builder, we needed to `push` the extension to Shoutem after creating a screen and shortcut to add them to app navigation. However, when we're only changing the app code, we don't need to `push`. Instead, we can use `shoutem link` to tell Shoutem CLI to bundle the local code of your extension.
+Let's preview your app again. We can preview it in the Builder, but it might take some time while Builder bundles the entire app again. Every time you change an extension, you'd have to _push_ it again and then the Builder would need to re-bundle the whole app to add the changes. It's much faster to use the **Shoutem Preview** app (available for [iOS]({{ site.shoutem.previewAppiOS }}) and [Android]({{ site.shoutem.previewAppAndroid }})) and the Shoutem CLI, which can bundle just the changes made in the extension instead of re-bundling the entire app, like the Builder would.
 
-```ShellSession
-$ shoutem link
-Extension successfully linked. Please, kill the packager before running the app.
-```
+You can also run your apps on local emulators the same way with regular React Native apps, we have a [tutorial]({{ site.url }}/docs/extensions/tutorials/setting-local-environment) that offers an in-depth explanation on all the ways you can develop locally.
```

Comment:
the same way with regular React Native apps (`react-native run-ios` / `react-native run-android`), ... What do you say to add that?